modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/Question.es.js

```diff
@@ -0,0 +1,157 @@
+import {ClayPaginationWithBasicItems} from '@clayui/pagination';
+import React, {useCallback, useEffect, useState} from 'react';
+
+import Answer from '../../components/Answer.es';
+import KeywordList from '../../components/KeywordList.es';
+import Rating from '../../components/Rating.es';
+import Subscription from '../../components/Subscription.es';
+import UserRow from '../../components/UserRow.es';
+import {createAnswer, getThread} from '../../utils/client.es';
+import {dateToInternationalHuman} from '../../utils/utils.es';
+
+export default ({
+					match: {
+						params: {questionId}
+					}
+				}) => {
+	const [answers, setAnswers] = useState([]);
+	const [articleBody, setArticleBody] = useState();
+	const [page, setPage] = useState(1);
+	const [question, setQuestion] = useState();
+
+	const loadThread = useCallback(
+		() =>
+			getThread(questionId, page).then(data => {
+				setQuestion(data);
+				setAnswers(data.messageBoardMessages.items);
+			}),
+		[page, questionId]
+	);
+
+	useEffect(() => {
+		loadThread();
+	}, [loadThread]);
+
+	const postAnswer = () => {
+		createAnswer(articleBody, question.id).then(() => {
+			setArticleBody('');
+			return loadThread();
+		});
+	};
+
+	return (
+		<section>
+			{question && (
+				<>
+					<div className="autofit-row autofit-padded">
+						<div className="autofit-col">
+							<Rating
+								aggregateRating={question.aggregateRating}
+								entityId={question.id}
+								myRating={question.myRating && question.myRating.ratingValue}
+								type={'Thread'}
+							/>
+						</div>
+
+						<div className="autofit-col autofit-col-expand">
+							<div className="autofit-section">
+								<h1>{question.headline}</h1>
+
+								<small>
+								<span>
+									Asked{' '}
+									{dateToInternationalHuman(
+										question.dateCreated
+									)}
+								</span>
+									<span>
+									{' '}
+										Active{' '}
+										{dateToInternationalHuman(
+											question.dateModified
+										)}
+								</span>
+									<span> Viewed {question.viewCount} times</span>
+								</small>
+
+								<Subscription
+									onSubscription={subscribed =>
+										setQuestion({...question, subscribed})
+									}
+									question={question}
+								/>
+
+								<p>{question.articleBody}</p>
+
+								<KeywordList keywords={question.keywords}/>
+							</div>
+							<UserRow answer={false} creator={question.creator}/>
+						</div>
+					</div>
+
+					<hr/>
+
+					<h3 className="subtitle">{answers.length} Answers</h3>
+
+					<hr/>
+
+					{answers.map(answer => (
+						<Answer answer={answer} key={answer.id}/>
+					))}
+
+					{!!answers.totalCount &&
+					 answers.totalCount > answers.pageSize && (
+						 <ClayPaginationWithBasicItems
+							 activePage={page}
+							 ellipsisBuffer={2}
+							 onPageChange={setPage}
+							 spritemap={
+								 Liferay.ThemeDisplay.getPathThemeImages() +
+								 '/lexicon/icons.svg'
+							 }
```

Comment:
Do you really need the spritemap?

Reply:
I thought that was needed to render the < > but it turns out, after testing it, that don't: https://github.com/boton/liferay-portal/pull/122/commits/65f9657d674b66492fcbe2cbe07522a80e847f42 :)